warehouse/templates/index.html

```diff
@@ -85,34 +90,30 @@ <h1 class="homepage-banner__title">Find, install and publish Python packages wit
 </section>
 
 <section class="horizontal-section horizontal-section--grey">
-  <div class="site-container">
-    <div class="col-half">
+  <div class="site-container" >
+    <div class="col-half" data-controller="show-trending-projects">
       <div class="heading-wsubtitle">
         <h2 class="heading-wsubtitle__heading">Trending Projects</h2>
         <p class="heading-wsubtitle__subtitle">Trending projects as downloaded by the community</p>
       </div>
       <div class="package-list">
         {% for release in trending_projects %}
-          {{ project_snippet(release) }}
+          {{ project_snippet(release, "trending") }}
         {% endfor %}
       </div>
-      {#
-      <a class="button" href="TODO">Show More</a>
-      #}
+      <button class="button" type="button" data-action="click->show-trending-projects#addFive" data-target="show-trending-projects.addFiveButton">Show More</button>
```

Comment:
remember to change the names on these, there won't be clashing as they're separate DOM elements and instances